src/components/Book.js

```diff
@@ -19,7 +19,7 @@ const Book = (props) => {
         <p className="author">{bookItem.author}</p>
       </div>
       <div className="buttons">
-        <button type="button" className="remove-btn" data-id={bookItem.id} onClick={removeBook}>
+        <button type="button" className="remove-btn" data-id={bookItem.item_id} onClick={removeBook}>
```

Comment:
- [ ] When I remove a book item, it does not leave until I refresh the page, this is not User friendly, Kindly fix